OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/ResetPasswordDialogFragment.java

```diff
@@ -30,14 +29,15 @@
 import retrofit2.Call;
 import roboguice.fragment.RoboDialogFragment;
 
+import static android.app.Activity.RESULT_OK;
+
 public class ResetPasswordDialogFragment extends RoboDialogFragment {
-    @Inject
-    private LoginService loginService;
+    private static final String ARG_LOGIN_EMAIL = "login_email";
 
-    private static final String EXTRA_LOGIN_EMAIL = "login_email";
+    public static final int REQUEST_CODE = 111;
```

Comment:
Since this is defined separately by this class, It would be better to have some randomly generated value like `0x573d705241AFDB82`, so that it would be unlikely to be duplicated by another class. Or we could have the Activity/Fragment provide it's own request code to this class as I suppose it's meant to be used, but then it would need to be stored and retained by this class. Either way is fine though I guess.